src/DssSpell.t.sol

```diff
@@ -146,23 +153,17 @@ contract DssSpellTest is DSTest, DSMath {
         return (expectedRate_ > yearlyYield_) ? expectedRate_ - yearlyYield_ : yearlyYield_ - expectedRate_;
     }
 
-    // Previous spell; supply if there is a need to test prior to its cast() function being called on mainnet.
-    SpellLike constant PREV_SPELL = SpellLike(0x8D602692eE4b5f0ec33A22fe6547822377FDCc4c);
-
-    // Time to warp to in order to allow the previous spell to be cast; ignored if PREV_SPELL is SpellLike(address(0)).
-    uint256   constant PREV_SPELL_EXECUTION_TIME = 1612108914;
-
     function castPreviousSpell() internal {
+        SpellLike _pspell = SpellLike(PREV_SPELL);
```

Comment:
If we're sticking with this pattern, could we name the variable `prevSpell` instead? Nowhere else in this file do we prefix local vars with underscores.

Reply:
done